src/test/scala/nl.knaw.dans.easy.deposit/TestSupportFixture.scala

```diff
@@ -57,14 +57,23 @@ trait TestSupportFixture extends FlatSpec with Matchers with Inside with BeforeA
 
   def minimalAppConfig: Configuration = {
     new Configuration("", new PropertiesConfiguration() {
-      private val draftDir: File = (testDir / "drafts")
-        .delete(true)
-        .createIfNotExists(asDirectory = true, createParents = true)
-      addProperty("deposits.drafts", draftDir.toString())
-      addProperty("pids.generator-service", "http://hostDoesNotExist")
+      addProperty("deposits.stage", testSubDir("stage").toString())
+      addProperty("deposits.drafts", testSubDir("drafts").toString())
+      addProperty("deposits.submit-to", testSubDir("easy-ingest-flow-inbox").toString())
```

Comment:
why don't you call this `testSubDir("submitTo")`?

Reply:
Was defined by @janvanmansum 

https://github.com/DANS-KNAW/easy-deposit-api/blame/dfc0a0b282a5af55fc7e054e27f4978658119352/src/main/assembly/dist/cfg/application.properties#L16-L18

Reply:
well, you link to production configuration, but here you're in the tests... So why does this have to match with the production configuration? During the UNIT tests it is not important how this project relates to or interacts with other projects.

Reply:
The following code would not work with your proposal

https://github.com/DANS-KNAW/easy-deposit-api/blob/0f9561036e02b71687d76184e0d4651ecc258f0d/src/test/scala/nl.knaw.dans.easy.deposit/servlets/IntegrationSpec.scala#L39-L41

https://github.com/DANS-KNAW/easy-deposit-api/blob/2d5c0c02a5bd906f36337aee066f3e01f1db75f7/src/test/scala/nl.knaw.dans.easy.deposit/servlets/DepositServletErrorSpec.scala#L34

https://github.com/DANS-KNAW/easy-deposit-api/blob/7c0364f67db0d4270130fd0e17a6da1af459e064/src/test/scala/nl.knaw.dans.easy.deposit/servlets/HappyRoutesSpec.scala#L34